src/relayclient/RelayClient.ts

```diff
@@ -440,6 +430,66 @@ export class RelayClient {
       message: `Transaction was not audited. Failed audit calls: ${failedAuditorsCount}/${auditors.length}`
     }
   }
+
+  getUnderlyingProvider (): HttpProvider {
```

Comment:
IWebProvider (aka ISendProvider)

Reply:
Fixed. We are having some conflicts here as we still have to cast to `HttpProvider` when passing the `RelayProvider` to the Web3 constructor.